cosmwasm/README.md

```diff
@@ -1,45 +1,92 @@
-# Deploy
+# Intro
 
-First build the contracts
+This folder contains pyth contract on CosmWasm and utilities to deploy it in CosmWasm chains.
 
+# Deployment
+
+Deploying a contract in terra consists of two steps:
+1. Uploading the code. This step will give you a code id.
+2. Optionally create a new contract or migrate an existing one:
```

Comment:
Are you required to do one or the other? If so, use "Either" instead of "optionally"

Reply:
Yes. Otherwise it's pointless :D 

Thanks.